.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
Did you mean to check in some of these files?   They probably should be added to the .gitignore (this and MANIFEST.in).